manifests/kustomize/base/pipeline/kustomization.yaml

```diff
@@ -26,4 +26,6 @@ resources:
 - ml-pipeline-viewer-crd-sa.yaml
 - pipeline-runner-role.yaml
 - pipeline-runner-rolebinding.yaml
-- pipeline-runner-sa.yaml
\ No newline at end of file
+- pipeline-runner-sa.yaml
```

Comment:
please add another folder under base/ named metadata for metadata components.

Reply:
Done.